src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig

```diff
@@ -161,27 +161,27 @@
       </div>
     </div>
 
-    {% if customerInformation.personalInformation.guest %}
-      {% form_theme transferGuestAccountForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
-
-      <div class="customer-guest-status row mb-1">
-        <div class="col-4 text-right">
-          {{ 'This customer is registered as a Guest.'|trans({}, 'Admin.Orderscustomers.Feature') }}
-        </div>
-        <div class="col-8">
-          {% if customerInformation.generalInformation.customerBySameEmailExists %}
-            <p>{{ 'A registered customer account using the defined email address already exists. '|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
-          {% else %}
-            {{ form_start(transferGuestAccountForm, {'action': path('admin_customers_transform_guest_to_customer', {'customerId': customerInformation.customerId.value})}) }}
-            {{ form_widget(transferGuestAccountForm.transfer_guest_account) }}
-
-            <p class="small-text">
-              {{ 'This feature generates a random password before sending an email to your customer.'|trans({}, 'Admin.Orderscustomers.Help') }}
-            </p>
-            {{ form_end(transferGuestAccountForm) }}
-          {% endif %}
-        </div>
+    <div class="customer-guest-status row mb-1">
+      <div class="col-sm-4 text-sm-right font-weight-bold">
+          {{ 'Customer type'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </div>
+      <div class="col-sm-8">
+        {% if customerInformation.personalInformation.guest %}
+            <p class="mb-0">{{ 'Guest customer'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
+            {% if customerInformation.generalInformation.customerBySameEmailExists %}
+              <p class="text-muted">{{ 'You can\'t transform this guest to a registered customer, because a registered customer with this email address already exists. '|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
+            {% else %}
+              <p class="text-muted">{{ 'You can transform this guest to a registered customer. The customer will receive an email with a link to set his password.'|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```

Comment:
```suggestion
              <p class="text-muted">{{ 'You can transform this guest to a registered customer. The customer will receive an email with a link to set the password.'|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```

probably better this way, wording has to be check with @l-delin

Reply:
I think "a" would be better here, as it's more neutral :-)